src/libraries/System.Threading.Tasks/tests/Task/TaskContinueWithTests.cs

```diff
@@ -295,6 +295,40 @@ public static void RunContinuationCancelTest()
             }
         }
 
+        // Test what happens when you cancel a task in the middle of a continuation chain.
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
+        public static void RunContinuationCancelTest_TimeSpan()
```

Comment:
Another one where it would be better to parameterize a single test rather than copy/paste the test and change one line.